onnxruntime/__init__.py

```diff
@@ -19,6 +19,11 @@
 # here we need to save the exception and continue with Cuda version validation in order to post
 # meaningful messages to the user.
 # the saved exception is raised after device version validation.
+try:
+    import torch
+except ImportError:
+    print("Torch not found. It is required for Eager mode.")
```

Comment:
It will affect all the users.  You are prompting onnx runtime inference users migrating from onnx runtime to pytorch. 
Onnx runtime inference package doesn't take pytorch as a dependency.